# Goal:
geometry_msgs/PoseStamped target_pose
string tool_frame
bool reset_controllers
#default True
float32 err_pos_goal
#default 0.02
float32 err_ang_goal
#default 0.35
float32 err_pos_max
#default 0.1
float32 err_ang_max
#default 1.0
uint32 steady_steps
#default 5
---
# Result:
string result
---
# Feedback message:
